src/status_im/ui/components/icons/icons.clj

```diff
@@ -4,9 +4,15 @@
 
 (def icon-path "./resources/images/icons/")
 
+(defn combine-path
+  [path el]
+  (if (System/getenv "COMPONENT_TEST")
+    (str path el "@2x.png")
+    (str path el ".png")))
+
 (defn require-icon
   [el]
-  (let [s (str "../resources/images/icons/" el ".png")
+  (let [s (combine-path "../resources/images/icons/" el)
```

Comment:
Same comment I made about the check against `js/global.__TEST__`, why do we need a different resource if the code is running via Jest? And have you tried using `with-redefs` in tests?

There's likely a better way, but maybe you've tried something already?

Reply:
oh, in reality this file will be removed once we remove all uses of the old icons. 

This code exists in the quo2/icons/icon.clj too, ideally we can configure this correctly so we don't need this logic within files but in the configuration files instead.

Reply:
I kind of understand, but which configuration files you mean would be ideal for setting up this?

Reply:
That's what I (we) need to figure out :)

Reply:
it's some asset loader, maybe it's in jest

Reply:
> That's what I (we) need to figure out :)

Okay, I see now. Do we have an issue to track this problem? I would be interested to help out when I finish some stuff this week.